### DISCUSSION TITLE
Image generation 768 fails

### DISCUSSION THREAD

**Discussion by vic-t** [Amblyopius/Stable-Diffusion-ONNX-FP16#27] [General]
Hi, 

I'm trying to get this running with a Radeon RX Vega 64 card with 8GB VRAM. I've followed the instructions by the letter and the 512x512 examples worked fine. When trying to run the 768x768 samples, the output creation seems to be progressing fast enough with about 1.4s/it but after 30/30 iterations, i.e. 100%, I get an error message from the the "AMD Bug Report Tool":

> AMD software detected that a driver timeout has occurred on your system. An issue report has been created....

I'm using AMD Driver Adrenalin Edition, 23.4.3. Should I maybe use the PRO version instead? Or is there something else to try?

**Comment by Amblyopius** [comment]
With 8GB of VRAM it probably slows down to a crawl when the VAE has to run, eventually leading to a time out. A lot of people try to fix this by raising the time out value but that won't fix the slowdown. Try to add --cpuvae as a parameter so the VAE runs on CPU. It runs slower on CPU than ideal circumstances on GPU but overall the generation will be faster when you hit VRAM limit.

**Comment by vic-t** [comment]
That helped - thanks!
